addons/interaction/initSettings.sqf

```diff
@@ -29,3 +29,11 @@
     format ["ACE %1", LLSTRING(DisplayName)],
     true
 ] call CBA_fnc_addSetting;
+
+[
+    QGVAR(interactWithTerrainObjects), "CHECKBOX",
+    "str_a3_modules_moduleomquest_defend_f_attributes_useterrainobject0",
+    format ["ACE %1", LLSTRING(DisplayName)],
```

Comment:
Should add a description that hints at what might go wrong with this.

Reply:
How about `Warning: some objects can collide with other objects.`?

Reply:
Works for me, pinging @mharis001.